homeassistant/components/reolink/host.py

```diff
@@ -386,10 +430,40 @@ def unregister_webhook(self):
         webhook.async_unregister(self._hass, self.webhook_id)
         self.webhook_id = None
 
+    async def _async_long_polling(self, *_) -> None:
+        """Use ONVIF long polling to immediately receive events."""
+        # This task will be cancelled once _async_stop_long_polling is called
+        while True:
+            if self._webhook_reachable:
+                self._long_poll_task = None
+                await self._async_stop_long_polling()
+                return
+
+            try:
+                channels = await self._api.pull_point_request()
+            except Exception as ex:  # pylint: disable=broad-except
```

Comment:
Does this need to be a broad exception catch? We should have control of the whole code path here so there shouldn't be any unknown exceptions unless something really goes wrong

Reply:
Indeed only ReolinkError would be expected (timeouts etc if network connection drops).
However since this is running as a seperate task, exceptions will not be logged so there would be no way of knowing or debuging if a unexpected error would occur.

I changed it to catch ReolinkError instead and bumped down the logging from exception to error.
I added a catch and re-raise for Exception that will log the exception.